src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -935,23 +927,12 @@ function logDebug(msg: string, data?: any): void {
   new EmulatorLog("DEBUG", "runtime-status", `[${process.pid}] ${msg}`, data).log();
 }
 
-async function invokeTrigger(
-  frb: FunctionsRuntimeBundle,
-  triggers: EmulatedTriggerMap
-): Promise<void> {
-  if (!frb.triggerId) {
-    throw new Error("frb.triggerId unexpectedly null");
-  }
-
+async function invokeTrigger(frb: FunctionsRuntimeBundle): Promise<void> {
   new EmulatorLog("INFO", "runtime-status", `Beginning execution of "${frb.triggerId}"`, {
     frb,
   }).log();
 
-  const trigger = triggers[frb.triggerId];
-  logDebug("triggerDefinition", trigger.definition);
-  const signature = getSignatureType(trigger.definition);
-
-  logDebug(`Running ${frb.triggerId} in signature ${signature}`);
+  logDebug(`Running ${frb.triggerId} in signature ${FUNCTION_TARGET_NAME}`);
```

Comment:
is FUNCTION_TARGET_NAME really the thing you want to print here as a "signature"?

Reply:
great catch!